packages/components/src/components/tabs/tabs-item/tabs-item-styles.ts

```diff
@@ -26,6 +26,10 @@ export const getComponentCss = (theme: Theme): string => {
           borderColor: 'transparent',
         },
       }),
+      // make content of tabs-item selectable
+      '::slotted(*)': addImportantToEachRule({
+        position: 'relative',
```

Comment:
Using `addImportantToRule()` on the single value would be faster.  
Also what about nested elements where the selector isn't applied on? Do they work?  
Perhaps this can be tested via e2e?